boards/riscv/adp_xc7k_ae350/adp_xc7k_ae350_defconfig

```diff
@@ -33,12 +33,21 @@ CONFIG_HWINFO_ANDES=y
 #CONFIG_FLOAT_HARD=y
 #CONFIG_DOUBLE_PRECISION_FPU=y
 
+# PMP granularity options
+CONFIG_PMP_GRANULARITY=8
+
 # HW DSP options
 #CONFIG_SOC_ANDES_V5_HWDSP=y
 
+# EXEC.IT options
+#CONFIG_SOC_ANDES_V5_EXECIT=y
```

Comment:
Why is this commented out?

Reply:
Only Andes toolchain support this option, disable this option for zephyr-sdk.